src/histogram.rs

```diff
@@ -264,15 +261,12 @@ impl Instant {
             // See more: https://linux.die.net/man/2/clock_gettime
             #[cfg(all(feature="nightly", target_os="linux"))]
             Instant::MonotonicCoarse(t) => {
-                const NANOS_PER_SEC: f64 = 1_000_000_000.0;
-                const NANOS_PER_MILLI: i64 = 1_000_000;
-                const MILLIS_PER_SEC: i64 = 1_000;
-
-                let now = get_time(CLOCK_MONOTONIC_COARSE as clock_t);
+                let now = get_time_coarse();
                 let now_ms = now.tv_sec * MILLIS_PER_SEC + now.tv_nsec / NANOS_PER_MILLI;
                 let t_ms = t.tv_sec * MILLIS_PER_SEC + t.tv_nsec / NANOS_PER_MILLI;
-                if now_ms >= t_ms {
-                    Duration::from_millis((now_ms - t_ms) as u64)
+                let dur = now_ms - t_ms;
+                if dur >= *CORASE_ERROR {
```

Comment:
What is the performance when checking these in multi-thread? It may be not effective to check with a global var frequently in this base library.

In some Ubuntu machines, the resolution is 4ms, I think this is too large.